contrib/nydus-snapshotter/pkg/filesystem/nydus/fs.go

```diff
@@ -234,6 +250,7 @@ func (fs *filesystem) NewDaemonConfig(labels map[string]string) (config.DaemonCo
 		return config.DaemonConfig{}, err
 	}
 
+	cfg.Device.Cache.Config.WorkDir = fs.cacheMgr.CacheDir()
```

Comment:
Better leave a comment here such as:
// Overriding `work_dir` option of nyudsd config as we want to set it via snapshotter config option to let snapshotter handle blob cache GC.

Reply:
done